Aptoide/aptoide-client-v8
# [ASV-1298] Reviewed all variables regarding marketName. Injected the …

**What does this PR do?**

   This PR aims to extract all usages of the marketName string to a variable.

**Database changed?**

   No

**Where should the reviewer start?**

- [x] Build.config

**How should this be manually tested?**

  Build the cobrand flavour, and check if the usages of the marketName string are all being correctly used.

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-1298



**Code Review Checklist**
